.eslintrc.js

```diff
@@ -31,6 +31,7 @@ module.exports = {
     "react/jsx-filename-extension": ["error", { extensions: [".tsx", ".jsx"] }],
     "jsdoc/newline-after-description": "off",
     "max-len": "off",
+    "no-await-in-loop": "off",
```

Comment:
we can get rid of this, right? it'll be one last lint rule to ignore in the code lol

Reply:
😄 yes please